src/sage/quadratic_forms/ternary_qf.py

```diff
@@ -241,8 +241,8 @@ def quadratic_form(self):
     def matrix(self):
         """
         Return the Hessian matrix associated to the ternary quadratic form.
-        That is, if `Q` is a ternary quadratic form, `Q(x,y,z) = a*x^2 + b*y^2 + c*z^2 + r*y*z + s*x*z + t*x*y`,
-        then the Hessian matrix associated to `Q` is
+        That is, if Q is a ternary quadratic form, Q`(x,y,z) = a*x^2 + b*y^2 + c*z^2 + r*y*z + s*x*z + t*x*y`,
+        then the Hessian matrix associated to Q is
         ::
```

Comment:
This is getting ridiculous, I should say. The original is perfect. Why make things bad?

Reply:
OK, I'll make all Qs math mode.